hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/healthcondition/HealthCondition.scala

```diff
@@ -0,0 +1,105 @@
+package org.broadinstitute.monster.dap.healthcondition
+
+import enumeratum.values.{LongEnum, LongEnumEntry}
+
+sealed abstract class HealthCondition(
+  override val value: Long,
+  val label: String,
+  val conditionType: HealthConditionType,
+  val abbreviation: String,
+  val hasCg: Boolean,
+  val hasDx: Boolean,
+  val isOther: Boolean = false,
+  val cgPrefixOverride: Option[String] = None,
+  val computeGate: String => String = identity
```

Comment:
These bottom two args are escape hatches for dealing with annoying cases. I'm not super happy with using them, but I figured it wasn't worth battling the terrible RedCap layout any longer.